webapp/api-doc/CreateLambdaInstance.md

```diff
@@ -0,0 +1,79 @@
+---
+title: API | create lambda instance
+description: create a new lambda instance
+---
+
+# API - create lambda instance - Description
+
+Lambda instance creation call. Given an authentication token through the header x-api-key, and the instance specifications through other HTTP headers, it will firstly check the validity of the token. If the token is invalid, the API will reply with a 401 error. If the token is valid, the API will start creating a new lambda instance, using the instance specifications specified via the HTTP headers. Then, the API will reply with the details of the cluster in creation, along with a 200 success code.
+
+## Basic Parameters
+
+Type | Description |
+-------|-----------------|
+ **Description** | create a new lambda instance
+ **URL**         | /backend/create_lambda_instance
+ **HTTP Method** | GET
+ **Security**    | Basic Authentication
+
+
+### Headers
+
+Type | Description | Required | Default value | Example value |
+------|-------------|----------|---------------|---------------|
+x-api-key | ~okeanos authentication token. If you have an account you may find the authentication token at (Dashboad-> API Access) https://accounts.okeanos.grnet.gr/ui/api_access. | `Yes` | None | tJ3b3f32f23ceuqdoS_TH7m0d6yxmlWL1r2ralKcttY
+x-auth-url | ~okeanos authentication url. If you have an account you may find the authentication url at (Dashboad-> API Access) https://accounts.okeanos.grnet.gr/ui/api_access. | `Yes` | None | https://accounts.okeanos.grnet.gr/identity/v2.0
+cloud-name | The cloud name specified in the .kamakirc file | `Yes` | None | lambda
```

Comment:
In this API call, we do not use any URL parameters. as all variables are passed as HTTP headers. Do we want an empty 'Parameters' section?